packages/@aws-cdk/aws-cloudformation/lib/pipeline-actions.ts

```diff
@@ -494,3 +502,16 @@ function stackArnFromProps(props: { stackName: string, region?: string }): strin
     resourceName: `${props.stackName}/*`
   });
 }
+
+function cfnCapabilities(adminPermissions: boolean, capabilities?: CloudFormationCapabilities): CloudFormationCapabilities {
+  if (adminPermissions && capabilities === undefined) {
+    // admin true default capability to NamedIAM
+    return CloudFormationCapabilities.NamedIAM;
+  } else if (capabilities === undefined) {
+    // else capabilities are undefined set AnonymousIAM
+    return CloudFormationCapabilities.AnonymousIAM;
+  } else {
+    // else capabilities are defined use them
+    return capabilities;
+  }
+}
```

Comment:
Sorry, maybe I wasn't clear before.

The capabilities should default to `AnonymousIAM` only in the `app-delivery` construct.

In the CFN Actions, the default should be `undefined` (that is, none), like it is now.

Reply:
I wondered about this change. With `adminPermissions` being required it seems a bit odd this default is allow IAM, but it's a pretty heavy coin flip considering it's rare not to deploy a role with a service.